libr/anal/var.c

```diff
@@ -192,6 +192,7 @@ R_API int r_anal_var_retype(RAnal *a, ut64 addr, int scope, int delta, char kind
 }
 
 R_API int r_anal_var_delete_all(RAnal *a, ut64 addr, const char kind) {
+	r_return_if_fail (a);
```

Comment:
you should use `r_return_val_if_fail (a, 0);`, because the function does need to return a value if the check `a` fails.